### PR TITLE
15 1j registration info precise

### PR DESCRIPTION
**Pull request by claraxuxu**

### CODE REVIEW

File: app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationInfoTest.kt [hunk 1]
```diff
@@ -0,0 +1,128 @@
+package com.bouboulecorp.thurii.RegistrationTests
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationInfoTest.kt:1]: 🚫 **[ktlint]** <final-newline> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File must end with a newline (\n)

File: app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationInfoTest.kt [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+import androidx.fragment.app.testing.launchFragmentInContainer
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationInfoTest.kt:6]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 3]
```diff
@@ -0,0 +1,132 @@
+package com.bouboulecorp.thurii.Registration.RegistrationFragment
+
+import android.app.DatePickerDialog
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:8]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import com.bouboulecorp.thurii.R
+import java.text.SimpleDateFormat
+import java.util.*
+
+class FragmentRegistrationInfo : Fragment(R.layout.registration_info) {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let{
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:18]: 🚫 **[ktlint]** <curly-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing before "{"

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+        // height
+        val heightAdapter = ArrayAdapter.createFromResource(
+            requireActivity(),
+            R.array.heights,
+            R.layout.spinner_item
+        )
+        val height = lay.findViewById<Spinner>(R.id.height_spinner)
+        height_listener(heightAdapter, height)
+
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:56]: 🚫 **[ktlint]** <no-blank-line-before-rbrace> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected blank line(s) before "}"

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+    }
+
+    fun birthday_listener(birthday: RelativeLayout, dayRes: TextView) {
+        birthday.setOnClickListener{
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:60]: 🚫 **[ktlint]** <curly-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing before "{"

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+            val getDate = Calendar.getInstance()
+            val datePicker = DatePickerDialog(
+                requireActivity(),
+                android.R.style.Theme_Holo_Light_Dialog_MinWidth,
+                DatePickerDialog.OnDateSetListener{
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:65]: 🚫 **[ktlint]** <curly-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing before "{"

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+                getDate.get(Calendar.YEAR),
+                getDate.get(Calendar.MONTH),
+                getDate.get(Calendar.DAY_OF_MONTH)
+            )
+            datePicker.show()
+        }
+    }
+
+    fun weight_listener(weightAdapter: ArrayAdapter<CharSequence>, weight: Spinner) {
+       weightAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:84]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (7) (should be 8)

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+        weight.adapter = weightAdapter
+        weight.onItemSelectedListener = object: AdapterView.OnItemSelectedListener {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:86]: 🚫 **[ktlint]** <colon-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing before ":"

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+            override fun onItemSelected(
+                parent: AdapterView<*>?, view: View?, position: Int, id: Long
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:88]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:88]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:88]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+        }
+    }
+
+    fun height_listener(heightAdapter: ArrayAdapter<CharSequence>, height: Spinner) {
+        val continue_btn = getActivity()?.findViewById<Button>(
+            R.id.continueBtn
+        )
+        heightAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
+        height.adapter = heightAdapter
+        height.onItemSelectedListener = object: AdapterView.OnItemSelectedListener {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:110]: 🚫 **[ktlint]** <colon-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing before ":"

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+            override fun onItemSelected(
+                parent: AdapterView<*>?, view: View?, position: Int, id: Long
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:112]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:112]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:112]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationInfoTest.kt [hunk 13]
```diff
@@ -0,0 +1,130 @@
+package com.bouboulecorp.thurii.RegistrationTests
+
+import androidx.fragment.app.testing.launchFragmentInContainer
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationInfoTest.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between

File: app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt [hunk 14]
```diff
@@ -0,0 +1,141 @@
+package com.bouboulecorp.thurii.Registration.RegistrationFragment
+
+import android.app.DatePickerDialog
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Registration/RegistrationFragment/FragmentRegistrationInfo.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between

**Review by larroyo3 — APPROVED** [review]
J'ai rajouté quelque taille mais tout le reste est bon :+1:

**Pull request merged**